TACC/Core-CMS
# fix: add+use core-styles.wysiwyg.css not copied code

## Overview

Create a `core-styles.wysiwyg.css` to load that code from installed Core-Styles (rather than rely on copied code).

## Related

- improves #595

## Changes

- **added** `core-styles.wysiwyg.css`
- **added** `.../ckeditor/contents.original.css` to have original contents of `.../ckeditor/contents.css`
- **changed** `.../ckeditor/contents.css` to load the two new stylesheets

## Testing

1. Create/Edit a Text plugin.
2. Style a block of text as "Island" (like in [#595](https://github.com/TACC/Core-CMS/pull/595) video).
3. Verify island is styled inside Text editor.

## UI

https://user-images.githubusercontent.com/62723358/230231879-f4474f0c-bf3d-4418-b7df-48e673243a03.mov
